detox/test/e2e/29.webview.test.js

```diff
@@ -1,194 +1,304 @@
+            await contentEditableElement.typeText('er', true);
 
-      //Initial Text
-      await editable.selectAllText();
-      await editable.typeText(MOCK_TEXT, true);
+            await expect(contentEditableElement).toHaveText('Name: Tester');
+          });
+        });
 
-      //Addition Text
-      const ADDITION_TEXT = ' AdditionText';
-      await editable.moveCursorToEnd();
-      await editable.typeText(ADDITION_TEXT, true);
+        it('should select all text in content-editable', async () => {
+          await contentEditableElement.selectAllText();
 
-      await expect(editable).toHaveText(MOCK_TEXT + ADDITION_TEXT);
-    });
-  });
+          await expectWebViewToMatchSnapshot('select-all-text-in-content-editable-webview');
+        });
 
-  it('should set input and change text', async () => {
-    // Verify initial value
-    const para = webview_1.element(by.web.id('testingPar'));
-    await expect(para).toHaveText('Message');
+        it('should focus on content-editable', async () => {
+          await contentEditableElement.focus();
 
-    const textInput = await webview_1.element(by.web.id('textInput'));
-    await textInput.scrollToView();
-    await textInput.tap();
-    await textInput.typeText(MOCK_TEXT);
+          await expectWebViewToMatchSnapshot('focus-on-content-editable-webview');
+        });
 
-    await webview_1.element(by.web.id('changeTextBtn')).tap();
-    // Verify text updated
-    await expect(para).toHaveText(MOCK_TEXT);
-  });
+        it('should move cursor to end', async () => {
+          await contentEditableElement.moveCursorToEnd();
 
-  it('should header get text and verify its value', async () => {
-    const text = await webview_1.element(by.web.id('testingh1')).getText();
+          await expectWebViewToMatchSnapshot('move-cursor-to-end-content-editable-webview');
+        });
+      });
 
-    const textInput = await webview_1.element(by.web.id('textInput'));
-    await textInput.scrollToView();
-    await textInput.tap();
-    await textInput.typeText(text);
+      it('should scroll to view', async () => {
+        await web.element(by.web.id('bottomParagraph')).scrollToView();
 
-    await webview_1.element(by.web.id('changeTextBtn')).tap();
+        await expectWebViewToMatchSnapshot('scroll-to-view-webview');
+      });
 
-    // Verify text is the title text
-    await expect(webview_1.element(by.web.id('testingPar'))).toHaveText(text);
+      it('should run script', async () => {
+        const headline = web.element(by.web.id('pageHeadline'));
+        await headline.runScript('(el) => { el.textContent = "Changed"; }');
 
-  });
+        await expect(headline).toHaveText('Changed');
+      });
 
-  it('should replace text', async () => {
-    const textInput = await webview_1.element(by.web.id('textInput'));
-    await textInput.scrollToView();
-    await textInput.tap();
-    await textInput.typeText('first text');
+      it('should run script with arguments', async () => {
```

Comment:
Why did you remove tests from scripts passed as regular functions? This will reduce test coverage.